packages/strapi-helper-plugin/lib/src/components/Select/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import ReactSelect, { components } from 'react-select';
+import PropTypes from 'prop-types';
+import { Carret } from '@buffetjs/icons';
+import { useTheme } from 'styled-components';
+import styles from './styles';
+
+const DropdownIndicator = props => {
+  const theme = useTheme();
+
+  return (
+    <components.DropdownIndicator {...props}>
+      <Carret fill={theme.main.colors.grey} />
+    </components.DropdownIndicator>
+  );
+};
+
+export const Select = ({ children, onChange, selectedValue, ...props }) => {
+  const childrenArray = React.Children.toArray(children);
+
+  const options = childrenArray.map(child => ({
+    value: child.props.value,
+    label: child.props.children,
+  }));
+
+  const selectedOption = options.find(({ value }) => value === selectedValue);
+
+  return (
+    <ReactSelect
+      options={options}
+      onChange={({ value }) => onChange(value)}
+      components={{ DropdownIndicator }}
+      styles={styles}
```

Comment:
Maybe we could pass the theme to the `style` prop. So we could use these values ​​in the style of the select.

```js
// Select/index.js
import { useTheme } from 'styled-components';

const theme = useTheme();

style={getSelectStyles(theme)}

// Select/getSelectStyles.js
const getSelectStyles = theme => ({
  ...
});
```

WYT about that?

Reply:
Yep sure 😊